gti.c

```diff
@@ -72,14 +72,18 @@ int main(int argc, char **argv)
     int i;
     char *git_path;
     (void) argc;
-
+    
+    char push[5];
+    memcpy(push, (*argv)+4, 4);
+    push[4] = '\0';
     open_term();
     TERM_WIDTH = term_width();
-    SLEEP_DELAY = 1000000 / (TERM_WIDTH + GTI_SPEED);
-
+    SLEEP_DELAY = 1300000 / (TERM_WIDTH + GTI_SPEED);
+   
     init_space();
-    for (i = -20; i < TERM_WIDTH; i++) {
-        draw_car(i);
+	for (i = -20; i < TERM_WIDTH; i++) {
+      if (!strcmp(push, "push")) push_car(i); 
+       else draw_car(i);
```

Comment:
Ha, so, the `push_car()` stuff is nice, but:
* why not strcmp with `argv[1]` directly? (but check argc before access!)
* the indentation here needs fixing
* don't change the `#defines` or the `SLEEP_DELAY`, or other unrelated stuff
* don't commit the binary
* I still don't understand the "opel" (or am I reading this wrong?). This is *clearly* a VW Golf GTI Mark 1. It even says so on the car. :D